test/tzdata/version.jl

```diff
@@ -29,11 +35,20 @@ end
 @test match(TZDATA_NEWS_REGEX, "Release 19999") === nothing
 
 
-archive = joinpath(ARCHIVE_DIR, "tzdata$TZDATA_VERSION.tar.gz")
+if VERSION >= v"1.4"
+    archive = @artifact_str "tzdata_$TZDATA_VERSION"
+else
+    archive = joinpath(ARCHIVE_DIR, "tzdata$TZDATA_VERSION.tar.gz")
+end
 
 mktempdir() do temp_dir
     # Read the first tzdata version
-    extract(archive, temp_dir, "NEWS")
+    if VERSION >= v"1.4"
+        temp_dir = archive  # so I don't need to have too many ifs
+    else
+        extract(archive, temp_dir, "NEWS")
+    end
```

Comment:
```suggestion
    if use_artifacts
        cp(joinpath(artifact_dir, "NEWS"), temp_dir)
    else
        extract(archive, temp_dir, "NEWS")
    end
```

Reply:
I needed to copy into NEWS directory explicitly, otherwise tests were failing, e.g. https://travis-ci.com/github/JuliaTime/TimeZones.jl/jobs/361061783